models/migrations/migrations.go

```diff
@@ -580,6 +580,8 @@ var migrations = []Migration{
 	NewMigration("Add unique index for project issue table", v1_23.AddUniqueIndexForProjectIssue),
 	// v295 -> v296
 	NewMigration("Add commit status summary table", v1_23.AddCommitStatusSummary),
+	// v296 -> v297
+	NewMigration("Add missed field of commit status summary table", v1_23.AddCommitStatusSummary2),
```

Comment:
```suggestion
	NewMigration("Add missing field of commit status summary table", v1_23.AddCommitStatusSummary2),
```

Better english. Also see [here](https://www.bbc.co.uk/worldservice/learningenglish/grammar/learnit/learnitv188.shtml).

Reply:
Not resolved.